test/e2e/router/debug_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package router
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/volcano-sh/kthena/test/e2e/framework"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func int64Ptr(i int64) *int64 {
+	return &i
+}
+
+// getRouterPod is a helper function to get the router pod
+func getRouterPod(t *testing.T, ctx context.Context, kthenaNamespace string) *corev1.Pod {
+	deployment, err := testCtx.KubeClient.AppsV1().Deployments(kthenaNamespace).Get(ctx, "kthena-router", metav1.GetOptions{})
+	require.NoError(t, err, "Failed to get router deployment")
+
+	// Build label selector from deployment selector
+	labelSelector := ""
+	for key, value := range deployment.Spec.Selector.MatchLabels {
+		if labelSelector != "" {
+			labelSelector += ","
+		}
+		labelSelector += key + "=" + value
+	}
+
+	// Get router pod
+	pods, err := testCtx.KubeClient.CoreV1().Pods(kthenaNamespace).List(ctx, metav1.ListOptions{
+		LabelSelector: labelSelector,
+	})
+	require.NoError(t, err, "Failed to list router pods")
+	require.NotEmpty(t, pods.Items, "No router pods found")
+
+	return &pods.Items[0]
+}
+
+// TestDebugServerPort tests that debug server is started with correct port configuration
+func TestDebugServerPort(t *testing.T) {
+	ctx := context.Background()
+	kthenaNamespace := framework.NewDefaultConfig().Namespace
+
+	routerPod := getRouterPod(t, ctx, kthenaNamespace)
+	t.Logf("Testing debug server on pod: %s/%s", routerPod.Namespace, routerPod.Name)
+
+	// Verify debug-port parameter is passed correctly
+	logs, err := testCtx.KubeClient.CoreV1().Pods(routerPod.Namespace).GetLogs(routerPod.Name, &corev1.PodLogOptions{
+		Container: "kthena-router",
+		TailLines: int64Ptr(100),
+	}).Stream(ctx)
+	require.NoError(t, err, "Failed to get pod logs")
+	defer logs.Close()
+
+	logBytes, err := io.ReadAll(logs)
+	require.NoError(t, err, "Failed to read pod logs")
+	logStr := string(logBytes)
+
+	// Verify debug-port flag is set correctly
+	assert.Contains(t, logStr, "Flag: debug-port, Value: 15000", "Debug port should be configured as 15000")
```

Comment:
donot do such test, it is quire inefficient, and you can test it via calling woth pod ip, which should fail